includes/admin/views/builder/video-explainer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Builder video explainer view
+ *
+ * @since [version]
+ * @version [version]
+ */
+?>
+<script type="text/html" id="tmpl-llms-video-explainer-template">
+
+	<span class="llms-video-explainer-trigger">
+		<img
+			src="<?php echo esc_url( plugin_dir_url( LLMS_PLUGIN_FILE ) . 'assets/images/course-builder-video-thumbnail.jpg' ); ?>"
+			alt="<?php esc_attr_e( 'How to Build Your Course Outline with the LifterLMS Course Builder', 'lifterlms' ); ?>"
+		/>
+	</span>
+
+	<div class="llms-video-explainer-wrapper">
+		<iframe
+			src="https://www.youtube.com/embed/kMd37cOsPIg"
+			class="llms-video-explainer-iframe"
+			title="<?php esc_attr_e( 'Build Your Course Outline with the LifterLMS Course Builder', 'lifterlms' ); ?>"
+			allow="accelerometer;autoplay;clipboard-write;encrypted-media;gyroscope;picture-in-picture;web-share"
+			allowfullscreen
+			loading="lazy"
```

Comment:
```suggestion
```
@seothemes I'm gonna remove this we'll see that later.
Basically, here, with that, when closing the pop up, and re-opening the iframe doesn't show up. Might be browser issue (chrome and firefox on mac) but still.